force-app/main/default/lwc/staffAvailability/staffAvailability.js

```diff
@@ -22,8 +23,11 @@ export default class StaffAvailability extends LightningElement {
 	@api statusColorNotAvailable;
 	@api statusColorAssigned;
 
-	namespace;
+	@track staffAvailability;
```

Comment:
boolean variable should start with a question, is, has, maybe use isContactAvailable?

Reply:
good catch, changing it now ^^